ssl/quic/quic_srtm.c

```diff
@@ -85,11 +91,20 @@ static unsigned long items_rev_hash(const SRTM_ITEM *item)
 static int items_rev_cmp(const SRTM_ITEM *a, const SRTM_ITEM *b)
 {
     /*
-     * We don't really need to use CRYPTO_memcmp here as the relationship of
-     * srt_blinded to srt is already cryptographically obfuscated, but it
-     * doesn't hurt.
+     * We don't need to use CRYPTO_memcmp here as the relationship of
+     * srt_blinded to srt is already cryptographically obfuscated.
      */
-    return CRYPTO_memcmp(a->srt_blinded, b->srt_blinded, sizeof(a->srt_blinded));
+    return memcmp(a->srt_blinded, b->srt_blinded, sizeof(a->srt_blinded));
```

Comment:
The comment still references CRYPTO_memcmp, but the call has just been changed to memcmp, was that intentional?